Configure

```diff
@@ -93,6 +93,7 @@ EOF
 # [no-]zlib     [don't] compile support for zlib compression.
 # zlib-dynamic  Like "zlib", but the zlib library is expected to be a shared
 #               library and will be loaded at run-time by the OpenSSL library.
+#               This option is not suggested for RPM based distributions.
```

Comment:
The whole comment somewhat confuses things as the zlib option itself can make libcrypto link dynamically to the zlib. So maybe something like:
```suggestion
# zlib-dynamic  Instead of linking the libcrypto to zlib library libcrypto
#               will dynamically load the shared zlib library at run-time.
#               This option requires the unversioned zlib shared library file
#               to be present on the system.
```